repository/src/main/java/org/alfresco/repo/security/authentication/identityservice/IdentityServiceFacadeFactoryBean.java

```diff
@@ -449,9 +511,19 @@ private Optional<OIDCProviderMetadata> extractMetadata(RestOperations rest, URI
 
         private Collection<URI> possibleMetadataURIs()
         {
-            return List.of(UriComponentsBuilder.fromUriString(config.getIssuerUrl())
-                                               .pathSegment(".well-known", "openid-configuration")
-                                               .build().toUri());
+            if (StringUtils.isBlank(config.getAuthServerUrl()) && StringUtils.isBlank(config.getIssuerUrl()))
+            {
+                throw new IdentityServiceException(
+                    "Failed to create ClientRegistration. The values of issuer url and auth server url cannot be empty.");

```

Comment:
```suggestion
                    "Failed to create ClientRegistration. The values of issuer url and auth server url cannot both be empty.");
```

I think it would make it clearer that only 1 of the 2 properties is necessary at least.

Reply:
Done.